--- ./conf-df3120/config_kernel.conf.minifs	2012-08-15 22:30:42.000000000 +0200
+++ ./conf-df3120/config_kernel.conf	2012-08-15 22:34:09.936965984 +0200
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.36.3
-# Mon Aug 13 22:34:36 2012
+# Wed Aug 15 22:34:09 2012
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -986,8 +986,7 @@
 # CONFIG_ACCESSIBILITY is not set
 CONFIG_RTC_LIB=y
 CONFIG_RTC_CLASS=y
-CONFIG_RTC_HCTOSYS=y
-CONFIG_RTC_HCTOSYS_DEVICE="rtc0"
+# CONFIG_RTC_HCTOSYS is not set
 # CONFIG_RTC_DEBUG is not set
 
 #
