--- ./build-df3120/linux.minifs/drivers/usb/gadget/s3c2410_udc.c	2012-08-14 16:29:32.000000000 +0200
+++ ./build-df3120/linux/drivers/usb/gadget/s3c2410_udc.c	2012-08-15 11:37:44.669278700 +0200
@@ -2033,6 +2033,12 @@
 
 	dprintk(DEBUG_NORMAL, "%s: version %s\n", gadget_name, DRIVER_VERSION);
 
+#ifdef CONFIG_USB_S3C2410_DEBUG_ON_DF3120
+
+	/* 2012-08-15 ufuchs@gmx.com 
+	 * if the debugfs isn't available the kernel prints this during startup
+	 * 'debugfs dir creation failed -19'
+	 */
 	s3c2410_udc_debugfs_root = debugfs_create_dir(gadget_name, NULL);
 	if (IS_ERR(s3c2410_udc_debugfs_root)) {
 		printk(KERN_ERR "%s: debugfs dir creation failed %ld\n",
@@ -2040,6 +2046,7 @@
 		s3c2410_udc_debugfs_root = NULL;
 	}
 
+#endif
 	retval = platform_driver_register(&udc_driver_2410);
 	if (retval)
 		goto err;
